app/routes/auth.py

```diff
@@ -13,6 +14,8 @@ def auth_login():
 
     return auth.set_user_jwt(json['token'], json.get('profile'))
 
-@server.route("/auth/logout", methods=['POST'])
+@server.route("/auth/logout", methods=['GET', 'POST'])
```

Comment:
`GET` is inappropriate for `logout`, because `GET` methods should not have side effects.